packages/ag-charts-enterprise/src/features/annotations/callout/calloutProperties.ts

```diff
@@ -28,4 +29,9 @@ export class CalloutProperties extends Fill(Stroke(TextualStartEndProperties)) {
                 return this.color;
         }
     }
+
+    override getPlaceholderColor() {
+        const textColor = this.color ?? '#888888';
+        return Color.mix(Color.fromString(textColor), Color.fromString(textColor), 0.66).toString();
```

Comment:
Does this do anything? It looks like it is just mixing the same color with itself?

Reply:
Makes it more transparent, I'm reusing the same util functions we already have to do it because the fills could be hex instead of rgb